modules/book.js

```diff
@@ -0,0 +1,20 @@
+const book = (bookArray) => {
+  const container = document.querySelector('.container');
+  if (bookArray.length > 0) {
+    bookArray.forEach((book) => {
+      const content = `<div class="book">
+                    <div class="left">
+                      <p>${book.title}</p>
+                      <p>by</p>
+                      <p>${book.author}</p>
+                    </div>
+                      <button class="remove">Remove</button>
+                  </div>`;
+      container.innerHTML += content;
+    });
+  } else {
+    container.innerHTML = '<h3 class="add-books">Add Books!</h3>';
+  }
```

Comment:
- [ ] The navbar links are not working. I'm not able to add books. Kindly fix this issue.